src/window.rs

```diff
@@ -360,6 +360,20 @@ pub enum WindowCreateError {
     Renderer/*(RendererError)*/,
 }
 
+impl_display! {
+    WindowCreateError,
+    {
+        DisplayCreateError(e) => format!("Could not create the display from the window and the EventsLoop: {}", e),
+        Gl(e) => format!("{}", e),
+        Context(e) => format!("{}", e),
+        CreateError(e) => format!("{}", e),
+        SwapBuffers(e) => format!("{}", e),
+        Io(e) => format!("{}", e),
+        WebGlNotSupported => "WebGl is not supported by webrender",
+        Renderer => "Webrender creation error (probably OpenGL missing?)"
```

Comment:
Nit: comma at the end missing

Reply:
Fixed. You might want to use `cargo fmt` by the way. It makes styling issues a lot easier for me as someone opening a pull request. Right now, if I use it, it changes almost the whole document. (That would not have solved this particular problem though, as `cargo fmt` does not affect macros).

Reply:
Hm, yeah, but don't use cargo fmt on this codebase without asking, please - yes, it makes the code "consistent", but often it makes it consistently unreadable. So I don't use it because in many cases I've found that it makes the code less readable - if a curly brace is on the same line or the next line doesn't make much of a difference when reading the code as long as you know what's going on but if you tell rustfmt to reformat a file and it changes all the tab alignment of certain struct, exploding one line to 10 with weird tabbing, just to stay under the 80 characters limit and as a result the code is now completely unreadable - that's where I start to question the benefits. 

This happened to me and as a result I had a lot of cleanup to manually correct rustfmts mistakes, which wasn't fun. I've also met other people who don't use rustfmt because of this... machines are great, but they can also backfire dramatically. Just a note.